src/main/java/seedu/address/logic/commands/DownloadAbstract.java

```diff
@@ -34,24 +34,28 @@
 
     protected static final String IVLE_LOGIN_BUTTON_ID = "ctl00_ctl00_ContentPlaceHolder1_btnSignIn";
 
-    protected static final String WRONG_PASS_USER_MESSAGE = "WRONG PASSWORD OR USERNAME ENTERED";
+    protected static final String MESSAGE_USERNAME_PASSWORD_ERROR = "WRONG PASSWORD OR USERNAME ENTERED";
 
     protected static final String IVLE_DOWNLOAD_PAGE_ADDRESS = "https://ivle.nus.edu.sg/v1/File/download_all.aspx";
 
     protected static final String IVLE_MODULE_LIST_FIELD_ID = "ctl00_ctl00_ctl00_ContentPlaceHolder1_ContentPlaceHolder1_ContentPlaceHolder1_ddlModule";
 
     protected static final String MESSAGE_MODULE_NOT_FOUND = "MODULE CODE NOT FOUND";
 
-    protected static final String MESSAGE_IVLE_LOGIN_FAIL = "UNABLE TO LOGIN TO IVLE AT THIS TIME";
+    protected static final String MESSAGE_UNABLE_REACH_IVLE = "UNABLE TO LOGIN TO IVLE AT THIS TIME";
```

Comment:
Similar formatting possible improvement.  This message should not all be in capital letters.